github/ipfs.yml

```
@@ -5048,13 +5048,14 @@ repositories:
     visibility: public
   helia-docker:
     advanced_security: false
-    allow_update_branch: false
+    allow_update_branch: true
     archived: false
     collaborators:
```

BTW, @whizzzkid, you can configure branch protections via GitHub Management like this even if you don't have admin access to a repo.

```suggestion
    branch_protection:
      main:
        allows_deletions: false
        allows_force_pushes: false
        blocks_creations: false
        enforce_admins: false
        lock_branch: false
        require_conversation_resolution: false
        require_signed_commits: false
        required_linear_history: false
        required_pull_request_reviews:
          dismiss_stale_reviews: false
          require_code_owner_reviews: false
          required_approving_review_count: 1
          restrict_dismissals: false
    collaborators:
```